### PR TITLE
+ Feedback.md

### PR DESCRIPTION
**Pull request by arasmussen**
Add Feedback.md with a link to Product Pains and an <iframe> that embeds a Product Pains widget inline for feedback specifically relevant to Redux.

### CODE REVIEW

**Comment by arasmussen** [comment]
Note that the iframe doesn't render in GitHub but should render fine on the actually docs website.

**Comment by arasmussen** [comment]
And you can see a demo of what the widget looks like at https://productpains.com/widget.html?token=07268479-03f5-d5b8-2626-0320223ff1ee :)

File: docs/Troubleshooting.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Troubleshooting
 
-This is a place to share common problems and solutions to them.
```
Comment (gaearon) [inline docs/Troubleshooting.md]: Spaces are significant in Markdown.

**Comment by arasmussen** [comment]
`git checkout HEAD^ docs/Troubleshooting.md`

**Comment by arasmussen** [comment]
(Removed whitespace changes, ready for another look)

**Comment by mxstbr** [comment]
Thanks very much @arasmussen!

**Comment by gaearon** [comment]
@mxstbr Have you tried publishing the docs btw? You should be able to, I think.

**Comment by gaearon** [comment]
I think `npm run docs:publish` should just work. If it doesn’t work for you let me know and we’ll figure it out!

**Pull request merged**